play-services-core/src/main/AndroidManifest.xml

```diff
@@ -570,8 +570,8 @@
 
         <activity
             android:name="org.microg.gms.auth.consent.ConsentSignInActivity"
-            android:process=":ui"
```

Comment:
You should keep the `:ui` process, otherwise the ui being closed can break things in the main services.

Reply:
Modified, but only UI thread processing will cause Message exception errors, so bundle is used to pass data